### DISCUSSION TITLE
Trial value based on max f1 score

### DISCUSSION THREAD

**Discussion by HelixPiano** [optuna/optuna#4827] [Q&A]
Hello everyone,
I am using Optuna to tune my model to achieve a better f1 score. Is there some way to use the best f1 score of the trial instead of the final one? Pretty new to optuna so I have still to learn a lot.

**Comment by nzw0301** [comment]
How about [`study.best_trial`](https://optuna.readthedocs.io/en/stable/reference/generated/optuna.study.Study.html#optuna.study.Study.best_trial)?
